packages/theming/apple-theme/README.md

```diff
@@ -0,0 +1,10 @@
+# Apple-theme
+
+Code and definitions for creating an Apple Theme for FluentUI React Native. Currently only implemented for macOS.
```

Comment:
Probably worth mentioning the bug tracking iOS progress

Reply:
Created Github #596 to track publicly